crates/core_arch/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Could you please add `.vscode` to `.gitignore` or `.git/ignore` (first gets pushed, the other doesn't)

Reply:
absolutely! Sorry about that!